atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/sweep/AbstractSweeperTest.java

```diff
@@ -127,8 +129,13 @@ protected static void setupTables(KeyValueService kvs) {
     }
 
     private static void tearDownTables(KeyValueService kvs) {
-        kvs.getAllTableNames().stream()
-                .forEach(tableRef -> kvs.dropTable(tableRef));
+        Awaitility.await()
+                .timeout(Duration.FIVE_MINUTES)
```

Comment:
How long does this take in normal circumstances? Is the five minute timeout chosen due to some kind of heartbeat timeout?